Tasks/Kirutsin/task4/task4.py

```diff
@@ -0,0 +1,48 @@
+# source
+index = 0
+final_string = ''
+user_choise = int(input('Hello!\nPlease provide maximum number of characters per line.Input please, your value:\n'))
+source = open('../../!Tasks/Task4/text.txt', 'r')
+text = source.read().replace('\n', '')
+format_string = ''
```

Comment:
where is any close instruction??? you just left the file hanging open, it's a nasty mistake